subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -1,7 +1,21 @@
 # SPDX-License-Identifier: Apache-2.0
 
+function(lib_inc_dirs)
+  target_include_directories(${ZEPHYR_CURRENT_LIBRARY} PRIVATE ${ARGN})
+endfunction()
```

Comment:
This makes the code less readable because the reader needs to learn a new function instead of just recognizing the familiar zephyr_library_include_directories.

Reply:
Revert this commit.